manifesto.md

```diff
@@ -21,7 +21,7 @@ We are a diverse team in every respect.
 
 ### No Meetings `#t`
 
-Get-togethers and [hackathons](./hackathons/README.md) only.
+Virtual Get-togethers, Show-&-Tells and [hackathons](./hackathons/README.md) only.
```

Comment:
```suggestion
(Virtual) Get-togethers, Show-&-Tells and [hackathons](./hackathons/README.md) only.
```

See the brackets as an expression of my hope that we will get out of our basements before we hit significant revenue.